sycl/test/basic_tests/vectors/swizzle.cpp

```diff
@@ -0,0 +1,43 @@
+// RUN: %clangxx -fsycl %s -o %t_default.out
+// RUN: %t_default.out
```

Comment:
Shouldn't SYCL program executed be tested by e2e tests?

Reply:
This is device agnostic and can run on the build system.